mergebot_poller.py

```diff
@@ -157,17 +169,6 @@ def merge_git(self, pull):
             True when successfully added. Return value is just to fulfill
             contract with search_github_pr since other commands could fail.
         """
-        _print_flush('Command was merge, adding to merge queue.')
+        self.l.info('Command was merge, adding to merge queue.')
```

Comment:
The use of self.l further contributes to confusion. Is there some way to have a single logger everywhere that is just called self.logger? Then it can be a different kind of logger (with appropriate methods that may log to 2 places) depending on the context?